### PR TITLE
service/dap: make stopped event description for exception more concise

### PR DESCRIPTION
**Pull request by suzmue**
This change removes "Paused on" in the stopped event description for panics, throws and runtime errors. We do not need to match the "Paused on reason" format that VS Code uses to display stopped events. For panics, throws, and runtime errors, "paused on" is actually not a useful description. Even if it is possible to continue executing the program, it is usually not going to be useful since it is crashing.

### CODE REVIEW

**Comment by suzmue** [comment]
Before:
Screen Shot 2021-06-02 at 5 30 33 PM

After:
Screen Shot 2021-06-02 at 5 30 57 PM

(Sorry about the long list of commits, will make sure to use a clean branch next time)

**Review by aarzilli — APPROVED** [review]
> LGTM, the commit description that github will generate will have to be cleaned up before merging, I think.

Or rebase-squash and force-push?

**Comment by derekparker** [comment]
Re-running CI.

**Review by derekparker — APPROVED** [review]
LGTM

**Comment by derekparker** [comment]
I want to wait on a +1 from @polinasok to merge.

**Review by polinasok — APPROVED** [review]
LGTM

**Comment by aarzilli** [comment]
Needs manual rebase.

**Pull request merged**
